### DISCUSSION TITLE
How to make pdfjs the default option when opening pdfs ?

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6203] [Q&A] [answered]
I love qutebrowser, it is an awesome project! But sometimes I miss common features/behavior of standard browser, such as clicking on a pdf url opens the pdf on a new tab without prompting the user for confirmation. Currently, if I click on a pdf url it asks me what to do (download, open it, etc) and if I select open with PDFJS then it opens two new tabs: open blank tab with url equals that of the pdf, and another tab where it displays the pdf. Is there a way to make Qutebrowser open the pdf with PDFJS by default and without opening the blank tab?

Thank you

**Comment by The-Compiler** [comment, marked as answer]
Enter `:set pdfjs` and take a guess :wink:

**Reply by uwla** [reply]
Thank you! This does do a good job. Still, it opens two tabs. One tab is a blank page and its url is the web URL of the pdf. The other tab is a preview of the PDF and the url is a temporary file on the system. Is there any way to not open the blank page?

**Reply by The-Compiler** [reply]
Not at the moment - relevant issues:

- [Use QWebEngineDownload::page() / open PDF.js in correct tab · Issue #4588 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/4588)
- [Download links open in new pages without their url. · Issue #1078 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/1078)
